src/client/utils/tracking/tracking.ts

```diff
@@ -69,7 +69,19 @@ export const { TrackAction, IdentifyAction } = setupTrackers<
   { debug: process.env.NODE_ENV === 'development' },
 )
 
-const adtractionProductMap: { [type in TypeOfContract]: number } = {
+const NOT_VALID_CONTRACTS = [
+  TypeOfContract.DkAccident,
+  TypeOfContract.DkAccidentStudent,
+  TypeOfContract.DkTravel,
+  TypeOfContract.DkTravelStudent,
+] as const
+type NotValidContracts = typeof NOT_VALID_CONTRACTS[number]
+
+type TypeOfContractExcludedUnused = Exclude<TypeOfContract, NotValidContracts>
+
+const adtractionProductMap: {
+  [type in TypeOfContractExcludedUnused]: number
+} = {
```

Comment:
Minor thing and this is optional, but I think it's easier to understand a Record :) 
```suggestion
const adtractionProductMap: Record<TypeOfContractExcludedUnused, number> = {
```

Reply:
I agree! ☝️